core/src/test/java/tech/tablesaw/filtering/deferred/DeferredColumnTest.java

```diff
@@ -0,0 +1,97 @@
+package tech.tablesaw.filtering.deferred;
+
+import org.junit.jupiter.api.Test;
+import tech.tablesaw.api.BooleanColumn;
+import tech.tablesaw.api.IntColumn;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.selection.BitmapBackedSelection;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static tech.tablesaw.filtering.deferred.QueryHelper.*;
```

Comment:
fwiw, I'm actually not a big fan of the `.*` imports. they're fine when I'm reading the code in my IDE. but when I'm reading it on GitHub it's hard to know where the methods are declared

I also don't think it's bad to make the test methods public. I don't care either way, but it is more common that they're public

Reply:
regarding the * imports, it depends. If you're writing production code, you shouldn't use them. If you're trying to make something that's useful for ad-hoc data analysis, you can't make people import every query option separately, and so we shouldn't have all our test code do it that way.  

Anyway, enjoy wherever you're off-to.

Reply:
Yeah, good point. I use them all the time for ad hoc code. I don't necessarily view our tests as being examples, but definitely see the argument for that in the actual `*Example.java` files. Anyway, it should be simple enough to turn that rule off